packages/amplify-cli/src/index.ts

```diff
@@ -64,20 +64,37 @@ process.on('unhandledRejection', function (error) {
   throw error;
 });
 
+function normalizeStatusCommandOptions( statusCommandOptions){
+  let options = statusCommandOptions;
+  //Normalize 'amplify status -v' to verbose, since -v is interpreted as 'version'
+  if ( options && options.hasOwnProperty('v') ){
+          options.verbose = true;
+          if ( typeof options['v'] === 'string'){
+            options[ options['v'] ] = true ;
+          }
+          delete options['v']
+  }
+  return options;
+}
+
 // entry from commandline
 export async function run() {
   try {
     deleteOldVersion();
 
     let pluginPlatform = await getPluginPlatform();
     let input = getCommandLineInput(pluginPlatform);
-
     // with non-help command supplied, give notification before execution
     if (input.command !== 'help') {
       // Checks for available update, defaults to a 1 day interval for notification
       notify({ defer: false, isGlobal: true });
     }
 
+    //Normalize status command options
+    if ( input.command == 'status'){
```

Comment:
Why do we need a special case for status? Can't we include this in the generic normalize parsing function?

Reply:
There were 2 data points for this decision
1. amplify -v flag maps to "version" of the library 
   If we were to support versioning ( and independent updating ) of categories in the future , we would need a "-v" option for each category, which would display, version, and the compatibility matrix.   
Status is special , since version information is provided already as part of the response (both summary and detailed).
2. We can definitely move this into a generic normalize function. The idea was to any specific behavior as a separate function , and to move them as specific implementations of an commandType interface .  
e.g <commandType>=><status>      => normalizeInput ( overridden by normalizeStatus ) 
                                    =><category> => normalizeInput ( overridden by category-specific